### DISCUSSION TITLE
Clip ignore Scenes (similar to column ignore scenes, but with an individual clip)

### DISCUSSION THREAD

**Discussion by FakenameMusic** [helgoboss/helgobox#1171] [Ideas]
Obviously Column ignores scenes is already a feature, but it would be great to be able to place non looped one-shots / risers / drum-fill clips in a column that is otherwise being used for something else. It would combine the existing functionality of clip layers (multiple audio sources per column) and give the ability to hit a one-shot at any time.

Why?
 It also means you don't have to create a duplicate Column of the same MIDI instrument or Wav source for one shot "fills" that don't follow scenes.. It allows a fill on an instrument to be on the same Scene row as the part of the song that it will be in, without being triggered by that Scene trigger at the start of the Scene, while allowing the rest of it's instances of that sound in the Column to behave in the expected way and follow scenes. So, you could have a clip that plays a little bass synth fill (at the end of your chorus before the verse starts again ) be stored in the same Scene as the chorus, but you can choose when to launch it, just before you want to move back to your next Verse scene where the bass will play as normal.

Allows "spare" empty clip slots in a Column to be used for fills / spot fx wavs etc without having to navigate to a different column using the up/down/left/right buttons on the controller. e.g. see below: I'd love to be able to use the clip circled top right as a one-shot white noise sweep without losing access to my kick drum on the left (red column) by having to page over to the next column. Lower in the column (below the circled one) I have a clip of a pad that comes in towards the end of the song on the bottom of that column (purple pad), but the column isn't being used at any other time in the song.

![image](https://github.com/user-attachments/assets/1b6e02e2-45b6-4b10-9ab4-1f443f12b13c)

**Comment by helgoboss** [comment]
This looks like a feature that would only be used with grid controllers, in order to build extremely individual control setups. In that case, you have an alternative: Remap a pad to a fixed slot via ReaLearn. Then it also ignores scrolling and appears "pinned".
